### PR TITLE
Harmonize log messages + include timeout value

### PR DESCRIPTION
**Pull request by spacegaier**
Some harmonization and spelling corrections plus inclusion of used timeout values in the log messages.

### CODE REVIEW

File: home-assistant-plugin/custom_components/xknx/__init__.py [hunk 1]
```diff
@@ -137,9 +137,9 @@ async def async_setup(hass, config):
         hass.data[DATA_XKNX].async_create_exposures()
         await hass.data[DATA_XKNX].start()
     except XKNXException as ex:
-        _LOGGER.warning("Can't connect to KNX interface: %s", ex)
+        _LOGGER.warning("Cannot connect to KNX interface: %s", ex)
```
Comment (marvin-w) [inline home-assistant-plugin/custom_components/xknx/__init__.py:140]: Can we use `Could not` or what was the intention of changing this?

**Review by marvin-w — CHANGES_REQUESTED** [review]

File: xknx/io/udp_client.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def error_received(self, exc):
         def connection_lost(self, exc):
             """Log error. Callback for connection lost."""
             if hasattr(self, 'xknx'):
-                self.xknx.logger.info('closing transport %s', exc)
+                self.xknx.logger.info('Closing transport: %s', exc)
```
Comment (marvin-w) [inline xknx/io/udp_client.py:67]: In my logs `exc` is always None. Maybe we can just remove the parameter? Or did you ever had this set to something else?

File: xknx/io/request_response.py [hunk 3]
```diff
@@ -67,8 +67,8 @@ def on_error_hook(self, knxipframe):
 
     def timeout(self):
         """Handle timeout for not having received expected knxipframe."""
-        self.xknx.logger.warning("Error: KNX bus did not respond in time to request of type '%s'",
-                                 self.__class__.__name__)
+        self.xknx.logger.warning("Error: KNX bus did not respond in time (%s secs) to request of type '%s'",
+                                 self.timeout_in_seconds, self.__class__.__name__)
```
Comment (marvin-w) [inline xknx/io/request_response.py:71]: Nice!

File: xknx/io/udp_client.py (hunk 2 above)
Comment (spacegaier) [reply xknx/io/udp_client.py:67]: Always "None" for me as well so far. But I wasn't sure if there might be other scenarios, so I did not touch that.

File: home-assistant-plugin/custom_components/xknx/__init__.py (hunk 1 above)
Comment (spacegaier) [reply home-assistant-plugin/custom_components/xknx/__init__.py:140]: "Could not" also works. I just wanted to get rid of the informal "Can't" and the typo "Cant" while I was already adjusting other messages.

Comment (marvin-w) [reply home-assistant-plugin/custom_components/xknx/__init__.py:140]: I personally don't care. If you want to change it again go ahead (I'd like that) but if not that's also no issue for me. In the end its the same.

File: xknx/io/udp_client.py (hunk 2 above)
Comment (marvin-w) [reply xknx/io/udp_client.py:67]: You can delete it IMHO

**Review by marvin-w — COMMENTED** [review]
Looks good! Thanks for your contribution!

File: xknx/io/udp_client.py (hunk 2 above)
Comment (farmio) [reply xknx/io/udp_client.py]: It is `None` when we close the connection or it reaches EOF. 
If the connection gets terminated from the other side there should be an exception.
We can explore this further when handling #349

**Pull request merged**